src/dataflow-types/src/client/controller.rs

```diff
@@ -328,8 +329,9 @@ where
         config: ConcreteComputeInstanceReplicaConfig,
     ) -> Result<(), anyhow::Error> {
         if let ConcreteComputeInstanceReplicaConfig::Managed {
-            size_config: _size_config,
-            availability_zone: _az,
+            size_config: _,
+            size_name: _,
+            availability_zone: _,
```

Comment:
Weak suggestion: Maybe `if let ConcreteComputeInstanceReplicaConfig::Managed { .. }` would be cleaner?

Reply:
@sploiselle's opinion is that we should list all fields here so developers are nudged into thinking about new fields affecting the below code (https://github.com/MaterializeInc/materialize/pull/12313#discussion_r869310618). I think this is reasonable, but I'd be fine either way.